src/SDK/Metrics/SdkCounter.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OpenTelemetry\SDK\Metrics;
+
+use OpenTelemetry\API\Metrics\Counter;
+use OpenTelemetry\SDK\Clock;
+
+final class SdkCounter implements Counter
```

Comment:
this and the following `SdkXxx` class names don't feel right. Should this be `class Counter implements CounterInterface`?